### PR TITLE
GL-531: Add result page, scenario Category 3

### PR DESCRIPTION
**Pull request by alexdesi**
### Jira link
https://transformuk.atlassian.net/browse/GL-531

### What?
Create new Result page only for the Cat3 scenario

### Why?
To provide info to the user that want to move goods to NI from UK, 
when Cat3 applies (not cat1 and cat2)

### Have you? (optional)

- [ ] Reviewed view changes with stake holders
- [ ] Validated mobile responsive behaviour of view changes

### Deployment risks (optional)
Low, this is behind Feature Flag

### CODE REVIEW

File: app/controllers/green_lanes/check_moving_requirements_controller.rb [hunk 1]
```diff
@@ -5,31 +5,71 @@ class CheckMovingRequirementsController < ApplicationController
...
+
+      if form.valid?
+        @goods_nomenclature = GreenLanes::GoodsNomenclature.find(
+          form.commodity_code,
+          {
+            filter: { geographical_area_id: form.country_of_origin },
+            as_of: form.moving_date,
+          },
+          { authorization: TradeTariffFrontend.green_lanes_api_token },
+        )
```
Comment (alexdesi) [inline app/controllers/green_lanes/check_moving_requirements_controller.rb]: Refactor: this validation could be moved into the form

Comment (alexdesi) [reply app/controllers/green_lanes/check_moving_requirements_controller.rb]: done

File: spec/requests/green_lanes/moving_requirements_controller_spec.rb [hunk 2]
```diff
@@ -52,14 +52,20 @@
         }
       end
 
-      it { is_expected.to redirect_to(result_green_lanes_check_moving_requirements_path) }
+      it do
+        expect(make_request).to redirect_to(result_green_lanes_check_moving_requirements_path(
```
Comment (jebw) [inline spec/requests/green_lanes/moving_requirements_controller_spec.rb:56]: Shouldn't this be testing the `response` object ?

**Review by jebw — APPROVED** [review]
LGTM

File: spec/requests/green_lanes/moving_requirements_controller_spec.rb (hunk 2 above)
Comment (alexdesi) [reply spec/requests/green_lanes/moving_requirements_controller_spec.rb:56]: I fixed the existing test, I think we are more interested in the redirection here.

**Review by shjohnson — APPROVED** [review]
LGTM

**Pull request merged**